domain/src/test/java/org/oppia/domain/exploration/ExplorationDataControllerTest.kt

```diff
@@ -165,6 +165,7 @@ class ExplorationDataControllerTest {
       )
 
       verify(mockExplorationObserver, atLeastOnce()).onChanged(explorationResultCaptor.capture())
+      explorationResultCaptor.value.getErrorOrNull()?.printStackTrace()
```

Comment:
Do we need it? or it's just you missed this line while testing this class?

Reply:
Whoops! Will delete that now!